.github/workflows/benchmark.yml

```diff
@@ -3,6 +3,10 @@ on:
   issue_comment:
     types: [created]
 
+concurrency:
+  group: plutus-benchmark
+  cancel-in-progress: true
```

Comment:
That seems wrong. If people make benchmark requests on multiple PRs, we want them to queue up and all get executed.

Reply:
Another reason why it's worth writing explanatory comments for stuff like this!